src/client/src/rt-components/popup/Popup.tsx

```diff
@@ -0,0 +1,63 @@
+import { darken } from 'polished'
+
+import React, { MouseEvent } from 'react'
+
+import { styled } from 'rt-theme'
+
+const PopupContainer = styled.div<{ open: boolean }>`
+  display: ${({ open }) => (open ? 'flex' : 'none')};
+  position: absolute;
+  z-index: 5;
+  align-items: center;
+  justify-content: center;
+  font-size: 0.75rem;
+  border-radius: 0.25rem;
+  background: ${({ theme }) => theme.core.alternateBackground};
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05),
+    0 1rem 3rem -1rem ${props => darken(0.1, props.theme.overlay.backgroundColor)};
+`
+
+const PopupPanel = styled.div`
```

Comment:
I think there is a semi-established pattern in parts of RTC to have a styled.tsx per component to contain styled components? I don't think its strict but I don't think it's a bad pattern to follow and I'm thinking that this component could be a candidate

Reply:
Agreed, doing it.

Reply:
Looking at the rt-components folder, none of them use this pattern. I'll start with the new Popup component!